src/standalone.ts

```diff
@@ -7,9 +8,25 @@ const aud = process.env.AUD || 'https://localhost:8443'
 
 const skipWac: boolean = !!process.env.SKIP_WAC
 
-const server = new Server(port, aud, skipWac)
+const tlsDir = process.env.TLS_DIR
+let httpsConfig
+if (tlsDir) {
+  httpsConfig = {
+    key: fs.readFileSync(`${tlsDir}/privkey.pem`),
+    cert: fs.readFileSync(`${tlsDir}/fullchain.pem`)
+  }
+}
+
+const owner = process.env.OWNER || 'https://michielbdejong.inrupt.net/profile/card#me'
```

Comment:
It probably should throw an error if you don't have an owner rather than defaulting to your webId.